lib/pages/portal/view/portal_page.dart

```diff
@@ -219,6 +220,17 @@ class _PortalPageState extends State<PortalPage> {
     return AppScaffold(
       title: Text(S.current.navigationPortal),
       actions: [
+        AppScaffoldAction(
```

Comment:
Unless you're actually seatching for websites (and at this point you aren't), put this button on the home page for now.